.env.sample

```diff
@@ -8,6 +8,5 @@ CARTODB_URL=https://myaccount.cartodb.com
 # DO_LOCAL_TMP_DIR=
 # DO_LOCAL_POSTGRESQL_10_LIB_DIR=
 
-# Default PG version
-# PGSERVICE # postgres/postgres10
+# Default PG port
```

Comment:
Although I'd drop PG9 support, I'd certainly keep the switch mechanism. The need will come from time to time, and since making the transition is not fast, we will be able to choose dinamically with this. What do you think? 

This also has impact on other changes of this PR. Probably, the `Makefile` won't have any hardcoded path or service identifier. In addition, the renamed files might not have to be renamed.